functions/templates/bulk-delivery-volunteer-email.md.mustache

```diff
@@ -1,74 +1,86 @@
-before coming to the warehouse.
-
-If you requested a separate shopping volunteer, please arrange to meet them at
-the store to pick up your additional items, with enough time to get to the
-warehouse by {{arrivalTime}}.
-
-The neighbors you're delivering to have confirmed their availability for
-1:30-4pm, but you'll call each of them before you leave the warehouse to get any
-last minute delivery details.
-
-You'll load your car with boxes/bags for the above ticket IDs, and then deliver
-them to the addresses below. Since there are perishables in the deliveries,
-you'll need to deliver them immediately after pickup. You may want to plan your
-route to Brooklyn Packers and then to the delivery locations in advance.
-
-If possible, we recommend printing this email out so you can mark tickets done
-as you complete them. Please also fill out the [Completion
-Form](https://airtable.com/shrvHf4k5lRo0I8F4) when you finish your deliveries.
-If any issues come up during your deliveries, or you are unable to deliver any
-of the boxes (because someone isn't home) contact Francesca at
-{{warehouseCoordinatorPhone}}. We'll help you redistribute the food to the
-community in another way.
-
-### Delivery Day Checklist
-
-- [ ] Shop for custom items (instructions sent separately) or meet your shopping
-      teammate at an agreed-upon time and place.
-
-- [ ] Check in with Hanna or Francesca at the warehouse when you arrive. They'll
-      let you know when your main food boxes and additional items are ready.
-      
-While you're waiting:
-
-- [ ] Call the recipients of each ticket to confirm someone will be home to
-      accept the delivery. (If they're not, please let Francesca or Hanna know --
-      we'll use their items for someone else, and deliver to them another time.)
-- [ ] At the warehouse, you'll collect the following for each household (we'll
-      give you more specific instructions day of):
-    - [ ] Main food boxes (may be multiple per household)
-    - [ ] Cleaning supplies
-    - [ ] Custom items
-    - [ ] Water
-- [ ] All boxes/bags will be labeled with ticket IDs. Please confirm all the
-      ticket IDs match, and have your route number/name on them.
-- [ ] Put everything in your car
-- [ ] Check off each delivery below as you complete it
-- [ ] Fill out the [delivery completion
-      form](https://airtable.com/shrvHf4k5lRo0I8F4) when you're done. If you
-      didn't do any shopping, specify $0 Total Cost.
+Thank you again for volunteering to help our neighbors.
+
+**We've assigned you Route {{routeName}} with the following tickets. We've
+listed your tickets in delivery order: {{ticketIDs}}**
+
+**Please arrive at Brooklyn Packers this Saturday, {{deliveryDateString}} at
+{{arrivalTime}} with your custom items already purchased.**
+
+Check out the instructions below. If any issues come up day-of, contact this
+week's Dispatcher: Annika at {{warehouseCoordinatorPhone1}}.
```

Comment:
would be cool if both phone number and name of dispatcher were params that could be passed in